docs/guide/static-deploy.md

```diff
@@ -187,6 +187,29 @@ The Netlify CLI will share with you a preview URL to inspect. When you are ready
 $ ntl deploy --prod
 ```
 
+## Cloudflare Pages
+
+1. Install [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/get-started/)
+2. Authenticate Wrangler with your Cloudflare account using `wrangler login`
+3. Run your build command 
+4. Deploy using `npx wrangler pages publish <Build-directory>`
+
+```bash
+# Install Wrangler CLI
+$ npm install -g wrangler
+
+# Login to Cloudflare account from CLI
+$ wrangler login
+
+# Run your build command 
+$ npm run build 
+
+# Create new deployment
+$ npx wrangler pages publish <Build-directory>
+
+```
```

Comment:
Let's remove the empty line above, and add an empty line below the triple backticks instead. GitHub doesn't allow me to suggest this.